### DISCUSSION TITLE
OSPOlogy Follow-up:  Ways to define a metrics strategy in your OSPO

### DISCUSSION THREAD

**Discussion by anajsana** [todogroup/ospology#194] [OSPO Stories]
Discuss with the community topics related to the upcoming OSPOlogy meeting:

📅 October, 5th ➡️ [Ways to define a metrics strategy in your OSPO](https://community.linuxfoundation.org/events/details/lfhq-todo-group-ospology-presents-ways-to-define-a-metrics-strategy-in-your-ospo/)

* Join [Zoom Link](https://zoom.us/j/96581498918?pwd=SnRhNmZaOWkzVUpmNHZKR3JBWlR1QT09)

In this open space, people can:

* Ask questions prior to the meeting starts
* Share notes from the conversation
* Make proposals for new initiatives that can help the OSPO Ecosystem
* Add useful links and resources related to the topic
* Connect with mentors involved in OSPOs and/or Open Source Management

📝 OSPOlogy Meeting Slides ➡️ [link](https://docs.google.com/presentation/d/1hr74s9MpOGUbulvQplGQZifzfwsjQjxZrqx8vOncsCs/edit?usp=sharing)

**Comment by anajsana** [comment]
Question coming from Q&A:

*Whenever we measure in a creative discipline like software development, it’s easy to lose context in “vanity metrics”. What kind of metrics would you call out as unimportant, much like measuring lines of code, numbers of commits, etc. from traditional software development?*

**Reply by GaryPWhite** [reply]
hey that's me! :)

**Reply by anajsana** [reply]
Answered by Sean Goggins:

*Metrics focused on projects and not individuals are useful. Individual metrics require a few tons of salt, and can easily be misused … comparing developers on different projects is not helpful.*

**Comment by anajsana** [comment]
Question coming from Q&A:

*In big organizations, how do you run a successful open source contribution strategy, how to involve the contributors on a consistent basis? Is it by choosing OSS projects which are consumed by the developers on a daily basis at work or by following other strategies?*

**Reply by anajsana** [reply]
Answered by Sean Goggins:

*Projects worth contributing to, in my opinion, are those that are critical to your profit-facing applications, and not market differentiating for your organization.*

**Comment by anajsana** [comment]
Question coming from Q&A:

*Hi Team, I wanted to understand how do we track the open source project score and looking at the score what all we can determine .. to suggest leaders that these are the strategies we as an organization need to follow ?*

**Reply by anajsana** [reply]
Answered by Sean:

*I think Daniel’s organization, Bitergia, brings over a decade of expertise. Bitergia’s tools are based on the CHAOSS Grimoirelab software. The CHAOSS Community is another resource, and there are other tools, like chaoss/augur as well.*

**Comment by anajsana** [comment]
Question coming from Q&A:

*There was some mention of dependencies earlier and other comments on ecosystem analysis. What is being provided to people? What are they doing with that information?*

**Reply by anajsana** [reply]
Answered by Sean:

*One organization I work with is a major employment website in the US, and they are using dependency analysis to understand which libraries they have dependencies on over a portfolio of ~11k projects. Looking across the portfolio at the biggest dependencies is a significant place where OSPOS add value.*

**Comment by anajsana** [comment]
Question coming from Q&A:

*Thoughts on providing context when a company has hundreds of projects to track. For instance - if you participate in Kubernetes vs. an important library that is largely stable and gets less attention but is also deeply important.*

**Reply by anajsana** [reply]
Answered by Sean:

*Because software involves humans, not relying exclusively on quantitative metrics is important. The qualitative information comes from talking with product managers and developers.*

**Comment by anajsana** [comment]
Question coming from Q&A:

*Where can that OSPO study and OSPO templates be found, please?*
